src/main/resources/checkstyle-suppressions.xml

```diff
@@ -28,6 +28,8 @@
     <suppress checks="." files=".*[\\/]?generated-sources[\\/].*"/>
 
     <!-- Exclude certain checks from test code -->
+    <suppress checks="AvoidStaticImport" files=".*Test\.java"/>
+    <suppress checks="AvoidStaticImport" files=".*TestPerf\.java"/>
     <suppress checks="JavadocVariable" files=".*Test\.java"/>
```

Comment:
Each of these should be a four-tuple; you should also exclude it for `.*IT\.java` and `.*ITPerf\.java`.